spm_BIDS.m

```diff
@@ -159,7 +159,7 @@
         
         %-Anatomy imaging data file
         %------------------------------------------------------------------
-        p = parse_filename(f{i}, {'sub','ses','acq','ce','rec','fa','echo','inv','run'});
+        p = parse_filename(f{i}, {'sub','ses','task','acq','ce','rec','run','echo','part','chunk'});
```

Comment:
This is missing the entities from quantitative MRI:
- mt 
- inv
- flip

https://bids-standard.github.io/bids-starter-kit/folders_and_files/files.html#anat-anatomical-mri

https://bids-specification.readthedocs.io/en/latest/appendices/qmri.html#inputs-are-file-collections